packages/base-driver/test/unit/helpers.js

```diff
@@ -0,0 +1,19 @@
+export function buildReqRes(url, method, body) {
```

Comment:
this may make more sense in `packages/driver-test-support/lib/helpers.js`

Reply:
ok. this made me write more types though :-(